crates/cargo-winrt/src/main.rs

```diff
@@ -317,7 +319,11 @@ fn try_download(url: String, recursion_amount: u8) -> anyhow::Result<Vec<u8>> {
         .url(&url)
         .map_err(|e| Error::DownloadError(e.into()))?;
     let status = &mut None;
+    let content_length = &mut None;
     {
+        if redirect {
+            handle.nobody(true)?;
```

Comment:
Can you explain what the advantage to setting nobody is here?

Reply:
When you try to get the header by the following code, actually it will get full content of download file without setting nobody to true.

```
let mut transfer = handle.transfer();
transfer
	.header_function(|header| {
		let header = std::str::from_utf8(header).unwrap();
		if header.starts_with("HTTP/1.1 ") {
			let n = &header[9..12];
			*status = Some(n.parse::<u16>().expect("Should be number"));
		}
		true
	})
	.unwrap();
print_verbose_status!("Requesting", &url);
transfer.perform()?;
```